src/components/core/DpEditor/DpEditor.vue

```diff
@@ -316,37 +314,43 @@
 import {
   Bold,
   BulletList,
+  Document,
   HardBreak,
   Heading,
   History,
   Italic,
   Link,
   ListItem,
   OrderedList,
+  Paragraph,
   Strike,
   Table,
   TableCell,
   TableHeader,
   TableRow,
+  Text,
   Underline
-} from 'tiptap-extensions'
+} from './libs/tiptapExtensions'
 import { CleanHtml, Tooltip } from '../../../directives'
 import {
   Editor, // Wrapper for prosemirror state
   EditorContent, // Renderless content element
-  EditorMenuBar // Renderless menubar
-} from 'tiptap'
-import { createSuggestion } from './libs/editorBuildSuggestion'
+} from '@tiptap/vue-2'
+// import {
+//   DpLinkModal,
+//   DpResizableImage,
+//   DpUploadModal,
+//   editorBuildSuggestion,
+//   EditorCustomDelete,
+//   EditorCustomImage,
+//   EditorCustomInsert,
+//   EditorCustomLink,
+//   EditorCustomMark,
+//   EditorInsertAtCursorPos,
+//   EditorObscure,
+//   Mention
+// } from './libs/tiptapExtensions'
```

Comment:
treu. But since its commented out it still works :-P 

```suggestion
// } from './libs/customExtensions'
```